etc/answers-core.api.md

```diff
@@ -96,6 +96,11 @@ export interface CombinedFilter {
 // @public
 export type Context = any;
 
+// @public
+export type CustomClientSdk = Record<string, string> & {

```

Comment:
Did the spec say to call it CustomClientSdk? I tried looking for the spec but I couldn't find it